src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -47,18 +47,18 @@ public void parseCommand_clear() throws Exception {
 
     @Test
     public void parseCommand_delete() throws Exception {
```

Comment:
Should this also be parseStudentCommand?

Reply:
Updated.